project/CONTRIBUTING.md

```diff
@@ -72,9 +72,33 @@ php-cs-fixer fix --verbose
 
 #### The documentation
 
-The documentation is mostly written with the `rst` format. You can test the doc rendering with the `make docs` command.
+The documentation is mostly written with the `rst` format, and can be found in the `Resources/doc` directory.
+You can test the doc rendering with the `make docs` command, but to do this, you will need [Sphinx][sphinx].
+Just like php dependencies can be managed with Composer, python dependencies can be managed with [pip][pip].
+To get sphinx, simply run the following command.
+
+```bash
+pip install --requirement {{ docs_path }}/requirements.txt --user
+```
+
+Some python binaries should be downloaded in `~/.local/bin`,
+[modify your `$PATH` environment variable](http://www.linfo.org/path_env_var.html)
+so that it contains this path and then, from the root of the project, run `make docs`
+
+If `make docs` is successful, you should be able to see your modifications:
+
+```bash
+$YOUR_FAVORITE_BROWSER {{ docs_path }}/_build/html/index.html
+```
 
 If your PR contains a new feature, you must add documentation for it.
+Of course, you can also create PRs consisting only in documentation changes.
+
+Documentation contributions should comply with [the Symfony documentation standards][sf-docs-standards]
+
+[sphinx]: http://www.sphinx-doc.org/en/stable/
+[pip]: https://pip.pypa.io/en/stable/installing/
+[sf-docs-standards]: https://symfony.com/doc/current/contributing/documentation/standards.html
```

Comment:
Please use underscores.

`sf-docs-standards` -> `symfony_docs_standards` (I was thinking you don't like shortened words :stuck_out_tongue: )